examples/deployment/scripts/transfer/gold_contract_sc.js

```diff
@@ -0,0 +1,34 @@
+const { transferMicroAlgosLsig } = require("algob");
```

Comment:
Please add a comment at the very beginning of the file explaining what the script is doing. This is an example, and we should make it very clear to the user what is happening here.

Reply:
Alright